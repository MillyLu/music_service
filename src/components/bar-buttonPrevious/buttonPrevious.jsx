import * as Styled from './styles';



export function ButtonPrevious({findPrevTrack}){
    return(
        <Styled.PlayerButtonPrev onClick={findPrevTrack}>
            <Styled.PlayerButtonPrevSvg alt='prev'>

            </Styled.PlayerButtonPrevSvg>
        </Styled.PlayerButtonPrev>
    )
}