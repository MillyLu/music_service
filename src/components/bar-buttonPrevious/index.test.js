/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { ButtonPrevious } from './buttonPrevious';



describe('<PlaylistContainer/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<ButtonPrevious />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });
