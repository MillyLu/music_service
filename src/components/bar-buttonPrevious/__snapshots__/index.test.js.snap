// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaylistContainer/> should render successfully and match snapshot 1`] = `
<Styled(styled.div)>
  <Styled(Component)
    alt="prev"
  />
</Styled(styled.div)>
`;
